cogs/induct.py

```diff
@@ -238,6 +237,21 @@ async def _perform_induction(self, ctx: TeXBotApplicationContext, induction_memb
                 reason=f"{ctx.user} used TeX Bot slash-command: \"/induct\"",
             )
 
+        intro_channel: discord.TextChannel | None = discord.utils.get(
+            self.bot.main_guild.text_channels,
+            name="introductions",
+        )
+
+        tex_emoji: discord.Emoji | None = self.bot.get_emoji(743218410409820213)
```

Comment:
Add a fallback check to attempt to get the TeX emoji from it's name if the ID no longer exists:

```python
if not tex_emoji:
    tex_emoji = discord.utils.get(main_guild.emojis, name="TeX")
```